ac_new.rb

```diff
@@ -119,7 +123,16 @@ def test_vconn1
       @driver.find_element(:id, 'pwd').send_keys ac_passwd
       @driver.find_element(:id, 'login-button').click
     end
-    @driver.get(@base_url + '/' + meeting_id + '?launcher=false&fcsContent=true&pbMode=normal')
+    @driver.get(@base_url + '/' + meeting_id + '?launcher=false&fcsContent=true&pbMode=normal&html-view=' + ac_html)
```

Comment:
I agree with the concern (not wanting to include a potentially unknown URL parameter), but I'm not sure there's a way to reliably detect when this parameter can be safely excluded from the URL. The issue is that older versions default to the Flash interface, whereas newer versions default to HTML5, which flips the requirements for including/excluding the URL param.

In theory we could treat the ENV var as specifying "this is the interface that my version of Adobe Connect uses," rather than "I am requesting this type of interface," and just assume that anyone using the older version wants to use the older interface and vice versa. But this seems insufficiently flexible, especially since there was a transition period where the same version of AC could display in either interface, and it seems worth retaining the ability to toggle back and forth.